### ISSUE TITLE
authentication module 

### ISSUE THREAD

**Issue by tchernobyl** [hookTracker/timeTrackerHook#2]
add authentication UI ,
you can see all APIs documentation in the postMan directory in the server app ;
use {{BaseUrl}}/auth/login end point to get the token

**Comment by tchernobyl** [comment]
@Ameur i think i'm done with the authentication handling ( can you check it ? ) but i've some questions :
do we have registration in timeTrackerHook ? or the user registration will be done through the Client TimeTracker Angular/React APP
after login success , we save the user token in the local Storage and redirect the user to his dashboard where he can see his profile and his company details , right ?

**Comment by tchernobyl** [comment]
@gdalyy yup we dont need registration form , it will be implemented in the admin application , it look good for me , just you have to recover the base url from the config file .
and start in the #3 issue

**Comment by gdalyy** [comment]
done
